packages/components/src/components/badges/badge.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { Badge } from './badge';
+
+describe('Badge', () => {
+	let element;
+	beforeEach(async () => {
+		element = new Badge()
+	});
+
+	it('should match snapshot', async () => {
+		const page = await newSpecPage({
+			components: [Badge],
+			html: `<t-badge>Label</t-badge>`,
+		});
+		expect(page.root).toMatchSnapshot();
+    });
+
+
+	it('should have a default css class', () => {
+		expect(element.getCssClassMap()).toBe('badge');
+	})
+
+	it("should have a link", async () => {
```

Comment:
Although you potentially can use a snapshot to test this feature it makes more sense to just test the logic with one unit test:
```
it('should handle link css class', () => {
		element.link = 'http://example-url.com'
		expect(element.getCssClassMap()).toContain('badge--link');
})
```

Reply:
done